client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -214,36 +193,41 @@ class RemoveDomainDialog extends Component {
 			return;
 		}
 		this.props.closeDialog();
-		this.setState( { step: 1 } );
+		this.setState( { domainValidated: false, step: 1 } );
 	};
 
 	render() {
 		const { purchase, translate, chatButton } = this.props;
 		const productName = getName( purchase );
+
+		// To be removed once "Never mind" is translated; the typo has been fixed in existing translations.
+		const closeDialogString = hasTranslation( 'Nevermind' )
+			? translate( 'Nevermind' )
+			: translate( 'Never mind' );
```

Comment:
I think this ternary is backwards. This says to use "Nevermind" if it's translated (which will always be true since it's already in this file), but I think you meant that you want to use "Never mind" if it's translated?

Also, I can't find the link right now but IIRC there's a bug where you can't use ternaries for translated strings... I could be wrong about that but it would be fairly easy to make this three statements instead of one (one variable for each translation and then a ternary to decide which one to use).

Reply:
Yeah, it looks backwards to me also...

I think the simplest way to deal with this might just be to use the "String Freeze" tag and then not deploy the change until translations are actually ready?

To avoid holding up the rest of the pull request on that, this typo fix could be moved to a separate followup also.

Reply:
Apparently String Freeze does not work for OSS PRs so I think using hasTranslation is a good option.